repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java

```diff
@@ -464,6 +464,20 @@ public EntityMutationResponse mapAttributesAndClassifications(EntityMutationCont
                         }
 
                         addHasLineage(inOutEdges, isRestoreEntity);
+                    } else {
+                        if (CollectionUtils.isNotEmpty(context.getEntitiesToRestore()) && context.getEntitiesToRestore().contains(vertex)) {
+                            Set<AtlasEdge> activeEdges = new HashSet<>();
+                            Iterator<AtlasEdge> iterator = vertex.getEdges(AtlasEdgeDirection.BOTH).iterator();
+                            while (iterator.hasNext()) {
+                                AtlasEdge edge = iterator.next();
+                                if (edge.getProperty(Constants.STATE_PROPERTY_KEY, String.class).equalsIgnoreCase("ACTIVE")) {
```

Comment:
Instead of hardcoding ACTIVE, we can use the pre-existing constant ACTIVE_STATE_VALUE

Reply:
Otherwise LGTM

Reply:
addressed thank you